app/internal/event.py

```diff
@@ -1,12 +1,73 @@
+import logging
 import re
+from typing import List
 
+from email_validator import validate_email, EmailSyntaxError
 from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
 from starlette.status import HTTP_400_BAD_REQUEST
 
+from app.database.models import Event
+
 ZOOM_REGEX = re.compile(r'https://.*?\.zoom.us/[a-z]/.[^.,\b\s]+')
 
 
-def validate_zoom_link(location):
+def raise_if_zoom_link_invalid(location):
     if ZOOM_REGEX.search(location) is None:
         raise HTTPException(status_code=HTTP_400_BAD_REQUEST,
                             detail="VC type with no valid zoom link")
+
+
+def get_invited_emails(invited_from_form: str):
+    invited_emails = []
+    for invited_email in invited_from_form.split(','):
+        invited_email = invited_email.strip()
+        try:
+            validate_email(invited_email, check_deliverability=False)
+            invited_emails.append(invited_email)
+        except EmailSyntaxError:
+            logging.error(f'{invited_email} is not a valid email address')
+
+    return invited_emails
+
+
+def get_uninvited_regular_emails(session: Session,
+                                 owner_id: int,
+                                 title: str,
+                                 invited_emails: List[str]):
+    invitees_query = session.query(Event).with_entities(Event.invitees)
+    similar_events_invitees = invitees_query.filter(Event.owner_id == owner_id,
+                                                    Event.title == title).all()
+    all_regular_invitees_concatenated = ''
+    for record in similar_events_invitees:
+        if record:
+            all_regular_invitees_concatenated += ',' + record[0]
```

Comment:
You join it and then split it. Can you make it more efficient? :)

Reply:
I don't see how I can do that without changing the models

Reply:
Just add a variable called `regular_invitees` before the `for` loop, and use `regular_invitees.add(record[0])` each iteration

Reply:
record[0] is a string of the format 'a@a.com,b@b.com'. I switched instead to ```regular_invitees.update(record[0].split(','))```